xarray/util/generate_aggregations.py

```diff
@@ -55,6 +56,23 @@ def reduce(
     ) -> {obj}:
         raise NotImplementedError()"""
 
+DEFAULT_CUMULATIVE_PREAMBLE = """
+
+class {obj}{cls}Cumulatives:
```

Comment:
```suggestion
DEFAULT_PREAMBLE = """

class {obj}{cls}Aggregations:
```
Rename the class to Aggregations and then having cumulatives in the same class should make sense.